logger.gd

```diff
@@ -432,7 +477,13 @@ func add_module(name, output_level = default_output_level, output_strategies = d
 func get_module(module = default_module_name):
 	"""Retrieve the given module if it exists; if not, it will be created."""
 	if not modules.has(module):
-		info("The requested module '%s' does not exist. It will be created with default values." % module, PLUGIN_NAME)
+		info(
+			(
+				"The requested module '%s' does not exist. It will be created with default values."
+				% module
+			),
+			PLUGIN_NAME
+		)
```

Comment:
IMO this kind of split doesn't really make things more readable. I'd argue it makes it worse.

Can we configure a higher max line length to avoid this wrappings?

Reply:
I've created an Issue https://github.com/Scony/godot-gdscript-toolkit/issues/305